py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -74,10 +75,17 @@ def _pretraced_backend(
             fake_mode, "allow_non_fake_inputs", True
         ), fake_mode:
             repair_input_aliasing(gm)
+
+            # Remove sym_int placeholders and inputs
+            remove_sym_nodes(gm)
+            torch_inputs = [
+                input for input in sample_inputs if isinstance(input, torch.Tensor)
+            ]
```

Comment:
Could `sample_inputs` in the function header be modified to `Sequence[Any]`, since symbolic ints would now also be valid inputs?